openmls/src/group/core_group/staged_commit.rs

```diff
@@ -352,6 +337,29 @@ impl CoreGroup {
             }
         }
     }
+
+    #[cfg(test)]
+    /// Helper function that reads the decryption keys from the key store
+    /// (unwrapping the result) and stages the given commit.
+    pub(crate) fn read_keys_and_stage_commit(
+        &self,
+        mls_content: &AuthenticatedContent,
+        proposal_store: &ProposalStore,
+        own_leaf_nodes: &[OpenMlsLeafNode],
+        backend: &impl OpenMlsCryptoProvider,
+    ) -> Result<StagedCommit, StageCommitError> {
+        let (old_epoch_keypairs, leaf_node_keypairs) = self
+            .read_decryption_keypairs(backend, own_leaf_nodes)
+            .unwrap();
```

Comment:
I know it's just a test function, but since you return a `Result`, why not use that instead of the `unwrap()`?

Reply:
Ah, my bad. I somehow thought it would return a `KeyStore::Error`, but since it's returning a `StageCommitError`, we can indeed just pass the error back up. Good catch!